scripts/quests/otherAreas/X_Marks_The_Spot.lua

```diff
@@ -0,0 +1,135 @@
+-----------------------------------
+-- X Marks the Spot
+-----------------------------------
+-- Log ID: 4, Quest ID: 65
+-----------------------------------
+-- CoP Mission 2-5 Ancient Vows Completed
+-----------------------------------
+-- !addmission 6 20
+-- Tavnazian Safehold       !zone 26
+-- Despachiaire             !pos 111.2090 -40.0148 -85.4810
+-- Parelbriaux              !pos 113.701 -41 42.3653
+-- Odeya                    !pos 83.3227 -34.25 -70.0384
+-- Phomiuna Aqueducts       !zone 27
+-- Hidden Hallway 2nd Gate  !pos 138.1027 -24 60.3209
+-----------------------------------
+
+local quest = Quest:new(xi.questLog.OTHER_AREAS, xi.quest.id.otherAreas.X_MARKS_THE_SPOT)
+
+quest.reward =
+{
+    gil = 4000,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == xi.questStatus.QUEST_AVAILABLE and
+                player:getCurrentMission(xi.mission.log_id.COP) >= xi.mission.id.cop.ANCIENT_VOWS
+        end,
+
+        [xi.zone.TAVNAZIAN_SAFEHOLD] =
+        {
+            ['Despachiaire'] = quest:progressEvent(144),
+
+            onEventFinish =
+            {
+                [144] = function(player, csid, option, npc)
+                    quest:begin(player)
+                end,
+            },
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == xi.questStatus.QUEST_ACCEPTED
+        end,
+
+        [xi.zone.TAVNAZIAN_SAFEHOLD] =
+        {
+            -- Will repeat until the quest is completed.
+            ['Despachiaire'] = quest:event(144),
```

Comment:
Good to not block, since there's a lot going on.  This _might_ be an importantOnce() though, since I'm seeing event 132 on repeated events.  We're treating 132 as importantEvent though, so it should at least cycle (might be good for a followup later with slanderous utterings active)